Makefile

```diff
@@ -88,7 +88,7 @@ endif
 	(cd test/e2e-tests/tests && S3_BUCKET=${S3_BUCKET} TARGET_IMAGE_TAGS_JSON=${TARGET_IMAGE_TAGS_JSON} NAMESPACE=${NAMESPACE} ${GO} test -timeout 30m)
 
 test:
-	@(./test/shell-pipeline-tests/seldon-bike-rentals/pipelines-test-seldon-bike-rentals.sh)
+#	@(./test/shell-pipeline-tests/seldon-bike-rentals/pipelines-test-seldon-bike-rentals.sh)
```

Comment:
Is the commenting out of this line expected to be committed, or was it just a local change while you were working on this?

Reply:
This is a part of the https://github.com/opendatahub-io/ai-edge/pull/243/commits/c6a399912fa21e7fe833032428c8063ecc9eedf2 commit which is here just for testing. It will be removed once PR check works.